example/spec/parallel/distributionAPI/distributionSpec.js

```diff
@@ -24,7 +24,7 @@ const s3Data = [
   '@cumulus/test-data/granules/MOD09GQ.A2016358.h13v04.006.2016360104606.hdf.met',
 ];
 
-describe('Distribution API', () => {
+xdescribe('Distribution API', () => {
```

Comment:
We want to skip these tests because they're TEA-specific. Again, we could delete.

Reply:
Yeah so this is a gray area because we support both TEA and Cognito but we only deploy Cognito _by default_ with the `example` deployments. So what if a change needs to happen involving the previous distribution API (with TEA)? In that case I would imagine the developer would want to deploy their stack with TEA and run this test suite against it. 

But then, is that likely to happen? Are we likely to have a change to TEA OR is the developer likely to remember this test suite? Maybe not. Maybe we remove these tests because the odds of needing them in the future are low enough. I'm conflicted. Thoughts? Am I thinking about this wrong?

Reply:
It feels like being in this limbo between Cumulus distribution and TEA is what complicates it. If we really are supporting both, then I guess just disabling this test is a fine approach for now. 

But overall it feels like technical debt we're holding on to "just in case"

Reply:
Yeah, I hear ya. I'd like to ping @laurenfrederick here if she has a second. I know the plan right now is to continue to support (and default to) TEA for the cumulus template deploys. Is there a plan to eventually deprecate TEA and the old distribution API in favor of the new one? If yes then we can add that context as a comment when we disable tests and stuff like this because we'll delete them eventually.

Otherwise, if we're always going to support both it does seem likely that we'll need a way to test TEA. That might mean that we need some better way to deploy and test both in our integration tests. Thoughts?

Reply:
It sounds like we're keeping both for now, so we can keep the tests